lib/mongodb.js

```diff
@@ -990,6 +990,8 @@ MongoDB.prototype.buildWhere = function(model, where, options) {
           g.warn('{{MongoDB}} regex syntax does not respect the {{`g`}} flag');
 
         query[k] = {$regex: cond};
+      } else if (spec === '$where') {
+        query[k] = {$where: cond};
```

Comment:
We use `query[k]['$' + spec] = cond;` to map LB operators to mongodb. Maybe we should check the `$` prefix to avoid `$$`.

Reply:
@raymondfeng I see good point.